### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Periareion**

### CODE REVIEW

**Comment by Periareion** [comment]
awesome opossom

File: SimpleTodo/todo/format.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ def to_string(cls,
         context: str,
         maths: str,
     ) -> str:
+
```
Comment (Qile0317) [inline SimpleTodo/todo/format.py:19]: why is there a random + here?

Comment (Qile0317) [reply SimpleTodo/todo/format.py:19]: ok ill just approve and change later

**Review by Qile0317 — COMMENTED** [review]
looks good to me

**Review by Qile0317 — APPROVED** [review]

**Pull request merged**